quickevent/app/quickevent/plugins/Runs/src/runflagsdialog.h

```diff
@@ -26,6 +26,8 @@ class RunFlagsDialog : public QDialog
 	Ui::RunFlagsDialog *ui;
 	RunsTableModel *m_model;
 	int m_row;
+	bool m_isDisqualified;
```

Comment:
no need for new class field, since `m_isDisqualified` can be computed anytime from checkboxes

Reply:
as there is a need to sync the state with the db entry  
i would argue this approach is more safe in case of future edits
```cpp
  table_row.setValue(QStringLiteral("runs.disqualified"), m_isDisqualified);
```

Reply:
what about `private bool isDisqualified() const;`?

Reply:
Seems good to me. :handshake:

Reply:
the best class field is no-field :)